srcpkgs/libpinyin/template

```diff
@@ -0,0 +1,43 @@
+# Template file for 'libpinyin'
+pkgname=libpinyin
+version=2.2.1
+revision=1
+build_style=gnu-configure
+hostmakedepends="intltool libtool pkg-config autoconf-archive autoconf automake wget gnome-common"
+makedepends="db-devel libglib-devel"
+short_desc="Support library for PinYin and ZhuYin (Bopofomo)"
+maintainer="Ben Sung Hsu <pobetiger+github@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/libpinyin/libpinyin"
+distfiles="https://github.com/libpinyin/libpinyin/archive/${version}.tar.gz"
+checksum=38a434f5f29607dcdf7c721cdff5a2428f03297a99064c7e15a9a64c1b831069
+
+# NOTE: nocross=yes
+#
+#    Data generation requires various binary from utils to be generated
+#    using build-system tools instead of the host (aka target) tools. The
+#    autotool project setup needs to be split up in order for cross compile to
+#    work.
+#
+#    Checking in this so at least x86 systems will have working build.
+#    Hope that in a future patch to we can figure out how to make the data
+#    generation binary to compile correctly.
+#
+#    NB. if compiled natively on aarch64/arm* systems using xbps-src,
+#    the compile will probably work just fine.
+nocross=yes
```

Comment:
just set nocross="Requires code generators to run as the host system but are compiled with target system"